inc/class-antivirus.php

```diff
@@ -350,10 +350,18 @@ public static function add_enqueue_style() {
 	/**
 	 * Get the currently activated theme.
 	 *
+	 * @param WP_Theme|false $theme Theme to parse.
+	 *
 	 * @return array|false An array holding the theme data or false on failure.
+	 *
+	 * @since 1.4 Added $t parameter.
 	 */
-	private static function _get_current_theme() {
-		$theme = wp_get_theme();
+	private static function _get_current_theme( $theme = false ) {
+		if ( false === $theme ) {
```

Comment:
`! $theme` should enough and faster. It's a bool value so no String compare is necessary.

Reply:
It may be `bool` (`false`) or `WP_Theme`.

Reply:
Yes, but the if statement checks only for a book value. Maybe a set to a bool value should helps, like (bool) $var. 
I would write code that always have the same typo of content, in this case always Boolean value, in a function.

Reply:
But that’s a whole different discussion. Could have used `null` here as well or rewrite the call to get around the fallback. However the `false` fallback is the most common variant across WP core.

`false ===` makes it explicit and has no potential implicit conversions. Although `WP_Theme` never evaluates to `false` in this specific case. Finally your initial comment is correct, the shorthand makes no difference here.

Reply:
Yes, I understand your view and is hard to validate the wp core there often give not the same type of return.